### PR TITLE
add base class for remote test to mock different environments

### PR DESCRIPTION
**Pull request by TomorrowIsAnOtherDay**

### CODE REVIEW

File: parl/remote/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+from parameterized import parameterized, parameterized_class
+from unittest import mock
+
+# How to add new configuration: provide turples with two arguements, where the first one represents the function be be mocked, and the second one is the mocked return.
```
Comment (zenghsh3) [inline parl/remote/test_utils.py]: turples -> tuples

Comment (zenghsh3) [reply parl/remote/test_utils.py]: arguements -> arguments

Comment (zenghsh3) [reply parl/remote/test_utils.py]: be be -> can be

**Review by zenghsh3 — CHANGES_REQUESTED** [review]

File: parl/remote/test_utils.py (hunk 1 above)
Comment (TomorrowIsAnOtherDay) [reply parl/remote/test_utils.py]: Sorry, I should check the spelling error before making a commitment.

**Review by zenghsh3 — APPROVED** [review]
LGTM

**Pull request merged**